### ISSUE TITLE
Wrong PostCSS version installed

### ISSUE THREAD

**Issue by hipstersmoothie** [vercel/next.js#23620]
### What version of Next.js are you using?

10.1.2

### What version of Node.js are you using?

14.16.0

### What browser are you using?

build issue

### What operating system are you using?

Mac Big Sur

### How are you deploying your application?

locally

### Describe the Bug

During a fresh install of `next.js` 2 versions of `postcss` are installed. `next.js` depends on v8 but v7 ends up in my node_modules

### Expected Behavior

Only 1 version of postcss is installed or the correct version is installed at the root

### To Reproduce

`npx create-next-app`

**Comment by kachkaev** [comment]
Looks like this issue is related to CVE-2021-23368. Snyk report: https://snyk.io/vuln/SNYK-JS-POSTCSS-1090595

```
yarn why next
=> Found "next@10.1.3"
```

```
yarn why postcss
=> Found "postcss@7.0.35"
   - Hoisted from "_project_#next#cssnano-simple#cssnano-preset-simple#postcss"
=> Found "next#postcss@8.1.7"
info This module exists because "_project_#next" depends on it.
```

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
